### PR TITLE
Add basic visibility options

### PR DESCRIPTION
**Pull request by eug-L**
* add visibility options and setoptions endpoint

### CODE REVIEW

File: admin/controller/module/tawkto.php [hunk 1]
```diff
@@ -128,13 +129,15 @@ private function getStoreHierarchy()
...
 			'current' => $this->getCurrentSettingsFor('0'),
+			'display_opts' => $this->getDisplayOpts('0'),
 		);
 
 		foreach ($stores as $store) {
 			$hierarchy[] = array(
 				'id'      => $store['store_id'],
 				'name'    => $store['name'],
 				'current' => $this->getCurrentSettingsFor($store['store_id']),
+				'display_opts' => $this->getDisplayOpts($store['store_id']),
```
Comment (pyhrus) [inline admin/controller/module/tawkto.php]: Could be improved to fetch from db once

File: admin/view/template/module/tawkto.twig [hunk 2]
```diff
@@ -174,5 +337,72 @@
 
 		});
 	}
+
+	jQuery(document).ready(function () {
```
Comment (pyhrus) [inline admin/view/template/module/tawkto.twig:338]: Could move 

// Init iframe
displayWidget();

into here

File: admin/view/template/module/tawkto.twig [hunk 3]
```diff
@@ -174,5 +337,72 @@
...
+								store.display_opts['show_oncustom'] = field.value.replaceAll('\r', '\n').split('\n').filter(Boolean);
+								continue;
+							}
+
+							if (field.name === 'hide_oncustom') {
+								store.display_opts['hide_oncustom'] = field.value.replaceAll('\r', '\n').split('\n').filter(Boolean);
+								continue;
+							}
+
+							store.display_opts[field.name] = field.value === '1';
```
Comment (pyhrus) [inline admin/view/template/module/tawkto.twig]: This value when render as json in storeHierarchy is boolean but updated as string '1'

**Review by pyhrus — APPROVED** [review]

File: admin/view/template/module/tawkto.twig (hunk 3 above)
Comment (eug-L) [reply admin/view/template/module/tawkto.twig]: serializeArray() only returns checked inputs, hence will change this to assign to true

**Pull request merged**